open_lm/main.py

```diff
@@ -212,6 +228,9 @@ def save_checkpoint(
             previous_checkpoint = os.path.join(args.checkpoint_path, f"optimizer_{completed_epoch - 1}.pt")
             if os.path.exists(previous_checkpoint):
                 os.remove(previous_checkpoint)
+            previous_checkpoint = os.path.join(args.checkpoint_path, f"stats_{completed_epoch - 1}.pt")
```

Comment:
Nit: can we make this a loop over epoch, optimizer and stats to avoid potential copy paste errors now and in the future?